sqldb-postgres/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "wasmcloud-provider-sqldb-postgres"
+version = "0.1.0"
+description = "Postgres SQL Database capability provider for contract wasmcloud:sqldb"
+readme = "README.md"
+edition = "2018"
+resolver = "2"
```

Comment:
We should add tags and categories. For tags we usually add at least wasmcloud and I think the category that might apply here is client library (on phone so can't look up the category slug for this atm)

Reply:
Ok I looked in https://crates.io/category_slugs and I think the 2 categories we need are wasm and database.

Reply:
we haven't been publishing capability providers to crates.io

Reply:
Touche. Move along, nothing to see here.